README.md

```diff
@@ -0,0 +1,22 @@
+# World Portals
+
+A module to add teleportation portals. Portals currently occupy a block and transport any entities that enter the occupied block to the portals destination world position.
+
+## Future Features
+
+- Block-linked portals that remove when the block is removed.
+- Cross world portals
+
+## PortalComponent
+A portal component represents a one way portal and has a block coordinate location and a world position destination.
+
+## PortalRasterizer & related.
+An abstract class that generates rectangular portal structures in the world, to be implemented in modules that need quick portals in the world or as a usage demo.
```

Comment:
I don't see any reference to any sort of "related" stuff here.

`or as a usage demo`
That is not something you explicitly need to mention.